### ISSUE TITLE
epubmeta produces "ERROR: FIXME with a better message" for some EPUB files

### ISSUE THREAD

**Issue by kingjon3377** [dino-/epub-tools#32]
When running `epubmeta /path/to/file.epub` with some EPUB files, it exits with exit code 1 and the output `ERROR: FIXME with a better message`. Of the EPUB files in my collection that trigger this (that are not obviously broken in other ways), about half were created by me using the [WebToEpub](https://github.com/dteviot/WebToEpub) browser extension, and the rest are downloaded from [Standard Ebooks](https://standardebooks.org/).

`epubmeta -e` doesn't give any error, and appears to work.

**Comment by dino-** [comment]
So, first of all, terrible error message on my part, I should be displaying some info about what happened. But this kind of immediate failure has never happened in the past with books I've tried and this area of the code got forgotten.
What it means is the book's metadata is completely unparseable by my code, which is wrong, these are valid books.
Looking into it, will fix when I have time.
Interestingly, [epubcheck](https://www.w3.org/publishing/epubcheck/) reports books I just downloaded from Standard Ebooks as EPUB version 3.3. I'll figure out what's going on with that and how to support it.
Thanks!
